test/Libraries/RevitIntegrationTests/ElementTests.cs

```diff
@@ -42,15 +44,32 @@ public void CanDeleteElement()
         /// Checks if Elements hosted elements can be retrived from Dynamo
         /// </summary>
         [Test]
-        [TestModel(@".\element.rvt")]
+        [TestModel(@".\Element\hostedElements.rvt")]
         public void CanGetHostedElements()
         {
-            string samplePath = Path.Combine(workingDirectory, @".\Element\GetHostElements_System_Test.dyn");
+            #region Arrange
```

Comment:
sorry if i wasn't clear, let's not use `region`s but simple comments `// Arrange`